docs/apis-tools/go-client/go-get-started.md

```diff
@@ -37,7 +37,7 @@ go mod init zb-user/zb-example
 2. To use the Zeebe Go client library, run the following:
 
 ```bash
-go get github.com/camunda/zeebe/clients/go/v8@v8.5.1
+go get github.com/camunda/camunda/clients/go/v8@v8.5.1
```

Comment:
❌ I guess this is for the upcoming version right? So we need to adjust the version as well.